eng/common/README.md

```diff
@@ -0,0 +1,28 @@
+# Don't touch this folder
```

Comment:
I'm ignoring everything under eng/common. To confirm, you just copied them from another repo❔

Reply:
Confirmed, this is a copy.